Sources/Extensions/SwiftStdlib/CollectionExtensions.swift

```diff
@@ -122,6 +122,21 @@ public extension Collection where Index == Int {
         return slices
     }
 
+    /// SwifterSwift: Returns the element at the specified position. If index is
+    /// negative, the nth element from the end will be returned. If index is out
+    /// of the indices, nil will be returned.
+    ///
+    ///        let arr = [1, 2, 3, 4, 5]
+    ///        arr[nth: 1] -> 2
+    ///        arr[nth: -2] -> 4
+    ///        arr[nth: -6] -> nil
+    ///
+    /// - Parameter index: The position of the element to access.
+    public subscript(nth index: Index) -> Element? {
```

Comment:
`nth` is not descriptive enough for a parameter name. I'm not sure what would be a better name, maybe `reverseIndex`, although you're using the index for both positive and negative values so `bidirectionalIndex`, although that may be a bit of a mouthful. Can you think of any suggestions?

Reply:
@guykogus I'm not sure, since `subscript` doesn't have auto-complete, i think the shorter the method name, the better.